ops/terraform/modules/stateless/main.tf

```diff
@@ -292,3 +292,23 @@ module "bfd_pipeline" {
     ci_cidrs      = [data.aws_vpc.mgmt.cidr_block]
   }
 }
+
+# Cloudwatch Log Metric Filters
+
+module "cw_metric_filters" {
+  source          = "../resources/cw_metric_filters"
+  env             = var.env_config.env
+}
+
+# Cloudwatch Log Metric Filter Alarms
+
+module "cw_metric_alarms" {
+  source          = "../resources/cw_metric_alarms"
+  env                           = var.env_config.env
+  app                           = "bfd"
```

Comment:
Doesn't really need to be a variable, as this wouldn't really be useable for any other app. No big deal, though.